meson.build

```diff
@@ -1,13 +1,13 @@
-project('spidermonkey-embedding-examples', 'cpp', version: 'esr90',
+project('spidermonkey-embedding-examples', 'cpp', version: 'esr91',
     meson_version: '>= 0.43.0',
-    default_options: ['cpp_std=c++17', 'warning_level=3'])
+    default_options: ['cpp_std=c++17', 'warning_level=1'])
```

Comment:
Does esr91 cause a bunch of new warnings? Maybe we should add exceptions lower down in `test_warning_args`?

Reply:
Oh, that wasn't supposed to stay in